### PR TITLE
Adds & updates Vaurca accents

### PR DESCRIPTION
**Pull request by Hockaa**
Adds three Vaurca accents: Zo'rane, K'laxane and C'thuric - sprites & descriptions provided by JamofBoy.

Also removes Biesel as the default accent of Vaurca and replaces it with TTS.

### CODE REVIEW

**Comment by Hockaa** [comment]
!review

File: code/modules/background/accent/vaurca.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+datum/accent/zora
```
Comment (NonQueueingMatt) [inline code/modules/background/accent/vaurca.dm:1]:
```suggestion
/datum/accent/zora
```

File: code/modules/background/accent/vaurca.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    name = ACCENT_ZORA
+    description = "Zo'rane is the modern day dialect of the Zo'ra hive, composed of buzzing, cheap bisellite voice modulators that manage to sway most voices \
+    into a more gender neutral tone, while retaining a distinct alien dialect. To other vaurcae, this accent is reminiscent of the authority the Zo'ra held - \
+    regal and noble. Their vocal augments stereotypically have difficulty, typically when producing sounds related to the letter 's', instead substituting with \
+    a harsh buzzing in the throat."
+    tag_icon = "zora"
+
+datum/accent/klax
```
Comment (NonQueueingMatt) [inline code/modules/background/accent/vaurca.dm:9]:
```suggestion
/datum/accent/klax
```

File: code/modules/background/accent/vaurca.dm [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    name = ACCENT_KLAX
+    description = "K'laxane is the modern day dialect of the K'lax hive. Primitive modulators, even among K'lax standards, manage to capture a similarity to the \
+    peasantry of the Izweski Hegemony on Moghes - however, they do not fully replicate it, retaining a more drawn out and dreamy tone. To other vaurcae, this is \
+    a more subservient manner of speaking, akin to how Bound tend to draw out their thoughts - a common trait instilled in most vassal hives as a brand. Their \
+    vocal augments stereotypically have difficulty, typically when producing sounds related to the letter 's', instead substituting with a harsh buzzing in the \
+    throat - the K'lax experience less of this - no Unathi would be content listening to a Vaura buzz incessantly when trying to hiss a sentence in the \
+    Hegemonic tongue."
+    tag_icon = "klax"
+
+datum/accent/cthur
```
Comment (NonQueueingMatt) [inline code/modules/background/accent/vaurca.dm:19]:
```suggestion
/datum/accent/cthur
```

File: code/modules/background/defines.dm [hunk 4]
```diff
@@ -90,4 +90,8 @@
 
 #define ACCENT_DIONA "Diona"
 
-#define ACCENT_SILICON "Text-to-Speech"
\ No newline at end of file
+#define ACCENT_SILICON "Text-to-Speech"
```
Comment (NonQueueingMatt) [inline code/modules/background/defines.dm:93]: This isn't very Silicon at this point, you should search  & replace it with ACCENT_TTS

**Pull request merged**